docs/content/components/select-menu.md

```diff
@@ -307,43 +307,38 @@ If the list is expected to get long, consider adding a `.SelectMenu-filter` inpu
         <h3 class="SelectMenu-title">Title</h3>
         <button class="SelectMenu-closeButton" type="button">
            <!-- <%= octicon "x" %> -->
-          <svg width="12" height="16" viewBox="0 0 12 16" class="octicon octicon-x" aria-hidden="true">
-            <path
-              fill-rule="evenodd"
-              d="M7.48 8l3.75 3.75-1.48 1.48L6 9.48l-3.75 3.75-1.48-1.48L4.52 8 .77 4.25l1.48-1.48L6 6.52l3.75-3.75 1.48 1.48L7.48 8z"
-            />
-          </svg>
+          <svg width="12" height="16" viewBox="0 0 12 16" class="octicon octicon-x" aria-hidden="true"><path fill-rule="evenodd" d="M7.48 8l3.75 3.75-1.48 1.48L6 9.48l-3.75 3.75-1.48-1.48L4.52 8 .77 4.25l1.48-1.48L6 6.52l3.75-3.75 1.48 1.48L7.48 8z" /></svg>
```

Comment:
This change I only added to make CI run again. But it's also nicer if the SVG doesn't take up too many lines.